test/Core.Test/AdminConsole/Services/PolicyServiceTests.cs

```diff
@@ -341,6 +360,75 @@ await sutProvider.GetDependency<IPolicyRepository>().Received()
         Assert.True(policy.RevisionDate - utcNow < TimeSpan.FromSeconds(1));
     }
 
+    [Theory, BitAutoData]
+    public async Task SaveAsync_ExistingPolicy_UpdateTwoFactor_WithoutMasterPasswordOr2FA_ThrowsBadRequest(
+        [AdminConsoleFixtures.Policy(PolicyType.TwoFactorAuthentication)] Policy policy, SutProvider<PolicyService> sutProvider)
+    {
+        // If the policy that this is updating isn't enabled then do some work now that the current one is enabled
```

Comment:
I don't understand this comment, is it still current?

Reply:
No, it was left from a copying and pasting another unit test.